Options.lua

```diff
@@ -5408,6 +5410,7 @@ do
                                     get = function() return true end,
                                     set = function() end,
                                     order = 0.02,
+                                    hidden = true,
```

Comment:
I like having this locked toggle in place (unhidden) to convey that your true actual (or lack of) target is always counted.